#!/usr/bin/env python
PACKAGE = "laser_detection_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("threshold", double_t, 0, "A double parameter",0, 0,4000)
gen.add("detector_id", int_t, 0, "detector_id", 0, 0,10)
gen.add("window_size", int_t, 0, "window_size", 60, 2,1000)
gen.add("weight", double_t, 0, "sensor_weight", 1, 1,10)
gen.add("reset",   bool_t,   0, "Reset Variable",  True)
gen.add("is_disable",   bool_t,   0, "Disable Sensor",  False)

exit(gen.generate(PACKAGE, "laser_detection_ros", "laser"))
n
